docs/src/main/asciidoc/kafka-streams.adoc

```diff
@@ -1113,6 +1113,28 @@ CMD ["./application", "-Dquarkus.http.host=0.0.0.0", "-Xmx32m"]
 Now start Docker Compose as described above
 (don't forget to rebuild the container images).
 
+== Kafka Streams Health Check
+
+If you are using the `quarkus-smallrye-health` extension, `quarkus-kafka-streams` will automatically add a readiness health check to validate that all topics declared in the `quarkus.kafka-streams.topics` property are created.
+
+So when you access the `/health/ready` endpoint of your application you will have information about the available and missing topics.
+
+This is an example of `DOWN` status:
+```
+{
+    "name": "Kafka Streams health check",
+    "status": "DOWN",
+    "data": {
+        "Available topic(s)": "weather-stations,temperature-values",
+        "Missing topic(s)": "hygrometry-values"
+    }
+}
+```
+If no topics are available, the `Available topic(s)` key will not be present in `data`.
+As well as if no topics are missing, the `Missing topic(s)` key will not be present in `data` and the `status` will be `UP`.
```

Comment:
> If no topics ..
I think this could be removed as it didn't add any usefull information.

Maybe this can be rephrased with something like:
> The check will be DOWN as soon as one topic is missing

Reply:
The meaning of these 2 sentences is : if the list of topics is empty, then the corresponding key is not in the body.

I precise this kind of information to ops guys that would like to set a kind of monitoring that parse the body of readiness.

Maybe the "and the `status` will be `UP`" part can be replace by you proposal before these two lines ?